src/Microsoft.Azure.WebJobs.Host/Abstractions/IActivityAbstraction.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host.Abstractions
+{
+    /// <summary>
+    /// Abstraction for Activity to avoid dependency on System.Diagnostics.DiagnosticSource
+    /// </summary>
+    public interface IActivityAbstraction : IDisposable
```

Comment:
Do we need this abstraction? `System.Diagnostics.Activity` should still be available without the package reference, right? It will just be an older version of it.